local/local.go

```diff
@@ -664,16 +748,24 @@ func inspectOptionalImage(docker client.CommonAPIClient, imageName string) (type
 	return inspect, nil
 }
 
-func defaultInspect(docker client.CommonAPIClient) (types.ImageInspect, error) {
-	daemonInfo, err := docker.Info(context.Background())
+func defaultInspect(platform imgutil.Platform) types.ImageInspect {
+	return types.ImageInspect{
+		Os:           platform.OS,
+		Architecture: platform.Architecture,
+		OsVersion:    platform.OSVersion,
+		Config:       &container.Config{},
```

Comment:
I we including fields like `Variant` and `Features` in `imgutil.Platform` and then ignoring them when they are set. Maybe we should just remove them?

Reply:
Removed all fields besides `Architecture`/`OS`/`OSVersion`